README.md

```diff
@@ -1,3 +1,5 @@
+# THIS REPOSITORY IS NO LONGER MAINTAINED AND ARCHIVED.
```

Comment:
This phrasing is confusing.

Reply:
The repository is no longer maintained and it is archived.  This is EOL.